integration_tests/src/main/python/orc_test.py

```diff
@@ -82,9 +82,16 @@ def test_basic_read(std_input_path, name, read_func, v1_enabled_list, orc_impl,
     StructGen([['child0', byte_gen], ['child1', orc_basic_struct_gen]]),
     StructGen([['child0', ArrayGen(short_gen)], ['child1', double_gen]])]
 
+all_basic_map_gens = [MapGen(f(nullable=False), f()) for f in [BooleanGen, ByteGen, ShortGen, IntegerGen, LongGen, FloatGen, DoubleGen, TimestampGen]] + [simple_string_to_string_map_gen]
+all_basic_map_gens += [DateGen(start=date(1590, 1, 1))]
+orc_map_gens_sample = all_basic_map_gens + [MapGen(StringGen(pattern='key_[0-9]', nullable=False), ArrayGen(string_gen), max_length=10),
+                   MapGen(RepeatSeqGen(IntegerGen(nullable=False), 10), long_gen, max_length=10),
+                   MapGen(StringGen(pattern='key_[0-9]', nullable=False), simple_string_to_string_map_gen)]
```

Comment:
Does your PR support Map(xx, Struct)? If yes, better add this for supporting.

Reply:
added map(struct, struct)